source/integrations/cloud-bot-accounts.rst

```diff
@@ -22,7 +22,7 @@ Note that currently:
 
   - Bot accounts can only be created or managed by plugins or System Administrators.
   - Bot accounts cannot be assigned to webhooks or slash commands. These must still be created by a user account.
-  - Service accounts without an email address pulled from LDAP or SAML systems is not yet supported.
+  - Service accounts without an email address pulled from LDAP or SAML systems are not yet supported.
```

Comment:
Perhaps there's an opportunity here to clarify this line further. On first read, it's not immediately clear what's intended. 
@justinegeffen - is it the case that service accounts with addresses pulled from other systems beyond LDAP or SAML systems are supported?

Reply:
Great question. I'll need to check on that and revert. For now we can accept this change.